Andrei_Tsimakhovich/02.GIT.Local/README.md

```diff
@@ -0,0 +1,58 @@
+#Homework Assignment 1: Initializing a Local Repository & Homework Assignment 7: Git Aliases and Configuration
+1. git init ./02.GIT.Local/
```

Comment:
all code blocks should be in codeblock section
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code